jobs/integration/utils.py

```diff
@@ -464,12 +468,13 @@ async def validate_storage_class(model, sc_name, test_name):
         log.info(f"output = {output.stdout}")
         assert "Hello, Storage!" in output.stdout
 
-    except asyncio.TimeoutError:
-        await _debug_ceph_storage(
-            control_plane=control_plane,
-            pods=[f"{sc_name}-write-test", f"{sc_name}-read-test"],
-        )
-
+    except:
+        # bare except intentional to debug any failure
+        if debug_open:
+            await _debug_storage_class(
+                debug_open, test_name, sc_name, provisioner, model
+            )
+        raise
```

Comment:
if we catch ANY error -- let's debug that error but don't continue the testing -- whichever exception needs to be re-raised so the test doesn't accidentally PASS